label_studio/storage/redis.py

```diff
@@ -0,0 +1,154 @@
+import json
+import os
+import logging
+import redis
+
+from .base import BaseStorage, BaseForm, StringField
+
+
+logger = logging.getLogger(__name__)
+
+def get_redis_connection(db = None):
+    if not db:
+        # This should never happen, but better to check than to accidentally 
+        # overwrite an existing database by choosing a wrong default:
+        logger.error("No redis db id passed!")
+    # Since tasks are always text, we use StrictRedis with utf-8 decoding.
+    return redis.StrictRedis(db=db, charset="utf-8", decode_responses=True)
+
+
+class RedisStorageForm(BaseForm):
+    path = StringField('Path', description='Storage prefix (optional)')
+    bound_params = dict(path='path')
+
+
+class RedisStorage(BaseStorage):
```

Comment:
Is the BaseStorage abstraction actually being used by project.Project? 

It doesn't appear to be.

Reply:
tbh, I just put it there because all the other classes implement it that way and I don't know the code well enough to just leave it out. Even if it doesn't, it might in the future...